playground/webpack.config.js

```diff
@@ -134,25 +134,23 @@ module.exports = (env = {production: false}) => ({
         test: /\.tsx?$/,
         use: [
           {
-            loader: 'awesome-typescript-loader',
+            loader: 'babel-loader',
+            options: {
+              babelrc: false,
+              minified: Boolean(env.production),
+              presets: [
+                ['shopify/web', {modules: false}],
+                ['shopify/react', {hot: true}],
+              ],
+              cacheDirectory: path.resolve(__dirname, 'build/cache/typescript'),
+            },
+          },
+          {
+            loader: 'ts-loader',
             options: {
               silent: true,
```

Comment:
why silent?

Reply:
We used silent before, and it's what SK does. It doesn't look like it has any effect on output as most errors are output via webpack rather than `console.log` per [their docs](https://github.com/TypeStrong/ts-loader#silent-boolean-defaultfalse)